packages/blade/src/components/Icons/Icons.stories.tsx

```diff
@@ -43,6 +43,51 @@ const Page = (): ReactElement => {
         export default App;
         `}
       </Sandbox>
+      <Title>Adding icons</Title>
+      <Description>
+        Please validate in the all icons story below in case the icon you wish to add is already
+        added. Steps for adding a new icon from Figma:
+      </Description>
+      <ul>
+        <li>
+          <Description>
+            Visit the [icons figma
+            file](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=57%3A0&t=xzz6HfFC50U8iuJp-0)
+          </Description>
+        </li>
+        <li>
+          <Description>
+            Click the icon you wish to add on Figma. Select the root icon element and not the layer
```

Comment:
```suggestion
            Click the icon you wish to add on Figma. Select the root icon element instead of the layer
```

Reply:
Done in https://github.com/razorpay/blade/pull/883/commits/6bd812011991d0077ccfdbc28db1625c68607145